Integrations/AWS_DynamoDB/AWS_DynamoDB.py

```diff
@@ -0,0 +1,1697 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+# flake8: noqa
+import boto3
+import json
+import datetime  # type: ignore
+from botocore.config import Config
+from botocore.parsers import ResponseParserError
+import urllib3.util
+
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+
+"""PARAMETERS"""
+AWS_DEFAULT_REGION = demisto.params().get('defaultRegion')
+AWS_ROLE_ARN = demisto.params().get('roleArn')
+AWS_ROLE_SESSION_NAME = demisto.params().get('roleSessionName')
+AWS_ROLE_SESSION_DURATION = demisto.params().get('sessionDuration')
+AWS_ROLE_POLICY = None
+AWS_ACCESS_KEY_ID = demisto.params().get('access_key')
+AWS_SECRET_ACCESS_KEY = demisto.params().get('secret_key')
+VERIFY_CERTIFICATE = not demisto.params().get('insecure', True)
+proxies = handle_proxy(proxy_param_name='proxy', checkbox_default_value=False)
+config = Config(
+    connect_timeout=1,
+    retries=dict(
+        max_attempts=5
+    ),
+    proxies=proxies
+)
+
+
+"""HELPER FUNCTIONS"""
+
+
+def safe_load_json(o):
+    kwargs = None
+    # Here we are checking the length of the entry to decide if it's a entry ID or json string.
+    if len(o) > 40:
+        try:
+            kwargs = json.loads(o)
+        except json.decoder.JSONDecodeError as e:
+            return_error(
+                'Unable to parse JSON string. Please verify the JSON is valid. - '+str(e))
+    else:
+        try:
+            path = demisto.getFilePath(o)
+            with open(path['path'], 'rb') as data:
+                try:
+                    kwargs = json.load(data)
+                except:  # lgtm [py/catch-base-exception]
+                    kwargs = json.loads(data.read())
+        except Exception as e:
+            return_error('Unable to parse JSON file. Please verify the JSON is valid or the Entry'
+                         'ID is correct. - '+str(e))
+    return kwargs
+
+
+def myconverter(o):
+    if isinstance(o, datetime.datetime):  # type: ignore
+        return o.__str__()
+
+
+def remove_empty_elements(d):
+    """recursively remove empty lists, empty dicts, or None elements from a dictionary"""
+
+    def empty(x):
+        return x is None or x == {} or x == []
+
+    if not isinstance(d, (dict, list)):
+        return d
+    elif isinstance(d, list):
+        return [v for v in (remove_empty_elements(v) for v in d) if not empty(v)]
+    else:
+        return {k: v for k, v in ((k, remove_empty_elements(v)) for k, v in d.items()) if not empty(v)}
+
+
+def aws_table_to_markdown(response, table_header):
+    if isinstance(response, dict):
+        if len(response) == 1:
+            if isinstance(response[list(response.keys())[0]], dict) or isinstance(response[list(response.keys())[0]], list):
```

Comment:
I don't know if it works but this could be rewritten in other way like:
```python
    if not isinstance(response, dict):
        return tableToMarkdown(table_header, response)
    elif len(response) != 1:
        return tableToMarkdown(table_header, response)
    elif not isinstance(response[list(response.keys())[0]], dict) and \
            not isinstance(response[list(response.keys())[0]], list):
        return tableToMarkdown(table_header, response)
    elif not isinstance(response[list(response.keys())[0]], list):
        return tableToMarkdown(table_header, response[list(response.keys())[0]])
    elif not isinstance(response[list(response.keys())[0]][0], str):
        return tableToMarkdown(table_header, response[list(response.keys())[0]])
    else:
        return tableToMarkdown(table_header, response[list(response.keys())[0]])
```